action.yml

```diff
@@ -13,7 +13,7 @@ inputs:
     type: number
     default: 1000
   log_file_prefix:
-    description: The prefix given to the log files that are uploaded
```

Comment:
The value is being overloaded and does two tasks here, but I think we should be fine as they're closely coupled?